src/datagrid/CustomFilter.tsx

```diff
@@ -53,7 +53,7 @@ export class CustomFilter extends React.PureComponent<CustomFilterProps, CustomF
     render() {
         const {customFilterValue} = this.state;
         return (
-            <div>
+            <div style={{width: "166px"}}>
```

Comment:
Observation - On using custom filter, after the data is filtered, the search box is not rendering as expected

Reply:
done